static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -170,7 +158,15 @@ function TagFacetsDistributionMeter({
           const unfocus = !!hoveredValue && hoveredValue.value !== segment.value;
           const focus = hoveredValue?.value === segment.value;
           return (
-            <Link key={`segment-${segment.name}-${index}`} to={segment.url}>
+            <Link
+              key={`segment-${segment.name}-${index}`}
+              to={segment.url}
+              aria-label={t(
+                'Add the %s %s segment tag to the search query',
```

Comment:
Should we edit this label? On Discover, it adds the tag to the search query but on Issue Details it takes you to the tag details tab

Reply:
This label will also be updated in this follow up pr: https://github.com/getsentry/sentry/pull/43646